src/mon/Monitor.cc

```diff
@@ -899,8 +899,6 @@ void Monitor::shutdown()
 
   wait_for_paxos_write();
 
-  state = STATE_SHUTDOWN;
-
```

Comment:
I don't think we can do this either :( because we use is_shutdown() in places like ms_dispatch to drop new work on the floor.  Without it, we will take in a start new work even after systems are shutting down..